### ISSUE TITLE
does not deploy QtQuick.Controls 1

### ISSUE THREAD

**Issue by sergii-koltsiuk** [probonopd/linuxdeployqt#180]
Look at [QTBUG-59054](https://bugreports.qt.io/browse/QTBUG-59054). We have the same problem.

**Comment by probonopd** [comment]
Yes, we also still seem to suffer from it, https://travis-ci.org/probonopd/linuxdeployqt/builds/300221680#L4524

> qrc:/main.qml:2 module "QtQuick.Controls" is not installed
